scss/civicrm/full-text-search/_full-text-search.scss

```diff
@@ -0,0 +1,19 @@
+.CRM_Contact_Form_Search_Custom .crm-search-form-block table.form-layout-compressed td {
+  padding: 2px 0;
+  width: auto;
+
+  label {
```

Comment:
Can solution be achieved without changing this? only padding change should do.

Reply:
or vertical align middle should do as well without changing any padding or margin